src/pipecat/services/aws_nova_sonic/aws.py

```diff
@@ -697,9 +697,9 @@ async def _receive_task_handler(self):
         try:
             while self._stream and not self._disconnecting:
                 output = await self._stream.await_output()
-                result = await output[1].receive()
+                result = await asyncio.wait_for(output[1].receive(), timeout=1.0)
```

Comment:
Why do we need the `asyncio.wait_for` in this case ?

Reply:
And if we do need it, I guess we would need to handle the `TimeoutError`.
But it feels like we shouldn't need the `asyncio.wait_for` here.

Reply:
Because `receive()` will wait until receiving the first item. So we need to reset the watchdog while we don't. Fixed.